README.md

```diff
@@ -220,6 +220,83 @@ You can also specify where the stylesheets are located with the `stylesdir` attr
 
     $ asciidoctor-pdf document.adoc -a stylesdir=css -a stylesheet="custom.css,override.css"
 
+**Asciidoctor extensions**
+
+Asciidoctor PDF can require Asciidoctor extensions written in JavaScript from the CLI.
```

Comment:
//require// is specific terminology for the Nodejs environment. Some end-users might not understand it.

Asciidoctor PDF can use Asciidoctor extensions which have been previously installed with node ?

Reply:
Initially, I wrote "can load" but it might also be too "technical".

> Asciidoctor PDF can use Asciidoctor extensions which have been previously installed with node ?

I like it :+1: 
I guess we should also update this sentence "It's also possible to require an extension from a JavaScript file." to "It's also possible to use an extension from a JavaScript file."?